src/Configuration.hpp

```diff
@@ -30,6 +30,9 @@ namespace imu_aceinna_openimu {
             ORIENTATION_AXIS right = ORIENTATION_AXIS_PLUS_Y;
             ORIENTATION_AXIS down = ORIENTATION_AXIS_PLUS_Z;
 
+            Orientation();
+            Orientation(ORIENTATION_AXIS forward, ORIENTATION_AXIS right, ORIENTATION_AXIS down);
```

Comment:
Some enums are named in upper `SNAKE_CASE` and some are in `PascalCase`. Our convention is the later, right?

Reply:
I think it is, yes. `UpperCamelCase` for the enum type, `UPPER_SNAKE_CASE` for the enum values. I'll fix it.

Reply:
Fixed.